tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -4741,7 +4755,7 @@ class HLSLExternalSource : public ExternalSemaSource {
       return true;
     case AR_OBJECT_TEXTURE_BUFFER:
```

Comment:
You fixed issues (presumably will fix Consume too) I pointed out in `GetHLSLResourceProperties()`, could you fix the Append/Consume cases above (both are UAVs) while you're at it?

Reply:
Just want to make sure I understand. The changes are to `ConsumeStructuredBuffer` in `GetHLSLResourceProperties` in DxilUtil.cpp and to the cases for `AR_OBJECT_CONSUME_STRUCTURED_BUFFER`  `AR_OBJECT_APPEND_STRUCTURED_BUFFER` here?